apps/ledger-live-desktop/src/newArch/Collectibles/Nfts/Collection/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { TableHeaderProps } from "../../types/Collection";
+import { useNftCollectionModel } from "./useNftCollectionModel";
+import { Box, Icons, Flex } from "@ledgerhq/react-ui";
+import TableContainer from "~/renderer/components/TableContainer";
+import TableHeader from "../../components/Collection/TableHeader";
+import { Account, ProtoNFT } from "@ledgerhq/types-live";
+import { useNftMetadata } from "@ledgerhq/live-nft-react";
+import TableRow from "../../components/Collection/TableRow";
+import EmptyCollection from "../../components/Collection/EmptyCollection";
+import { CollectibleTypeEnum } from "../../types/Collectibles";
+import Button from "~/renderer/components/Button";
+import { useTranslation } from "react-i18next";
+import ShowMore from "../../components/Collection/ShowMore";
+import { FieldStatus } from "../../types/DetailDrawer";
+import CollectionContextMenu from "LLD/components/ContextMenu/CollectibleContextMenu";
+
+export type NftsInTheCollections = {
+  contract: string;
+  nft: ProtoNFT;
+  nftsNumber: number;
+  onClick: (collectionAddress: string) => void;
+};
+
+type ViewProps = {
+  tableHeaderProps: TableHeaderProps;
+  nftsInTheCollection: NftsInTheCollections[];
+  account: Account;
+  displayShowMore: boolean;
+  onReceive: () => void;
+  onOpenCollection: (collectionAddress: string) => void;
+  onShowMore: () => void;
+};
+
+type NftItemProps = {
+  contract: string;
+  tokenId: string;
+  currencyId: string;
+  numberOfNfts: number;
+  account: Account;
+  onClick: (collectionAddress: string) => void;
+};
+
+type Props = {
+  account: Account;
+};
+
+const NftItem: React.FC<NftItemProps> = ({
+  contract,
+  tokenId,
+  currencyId,
+  account,
+  numberOfNfts,
+  onClick,
+}) => {
+  const { metadata, status } = useNftMetadata(contract, tokenId, currencyId);
+
```

Comment:
as you use several times `status === FieldStatus.Loading` you can use a const

Reply:
Done ✅